geomesa-features/geomesa-feature-avro-complex/pom.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>geomesa-features</artifactId>
+        <groupId>org.locationtech.geomesa</groupId>
+        <version>1.2.2-SNAPSHOT</version>
```

Comment:
GeoMesa 1.2.4 was released last night.  Do we need this to be on the 1.2.2-SNAPSHOT, or can we upgrade?